lib/framework/notification_manager.dart

```diff
@@ -124,7 +130,8 @@ class NotificationManager {
   }
 
   Future<void> _handleNotification() async {
-    Map<String, dynamic>? messageData = Ensemble.externalDataContext['data'];
+    Map<String, dynamic>? messageData =
```

Comment:
why do you have to read the data from externalDataContext? What is externalDataContext? The RemoteMessage is already there.